dateparser/conf.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+class Settings(object):
+    PREFER_DATES_FROM_FUTURE = None
```

Comment:
It is very confusing when option have both `None` and `False` as valid options. It is best to use here two different `True/False` options and add some validation, so they both could not be True.